buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -601,6 +601,10 @@
       "value": "symbolpkg\\*\\*\\*.nupkg",
       "allowOverride": true
     },
+    "SymbolPackageLocation": {
```

Comment:
Do these globs not work with `symbolpkg\\**\\*.nupkg` syntax?

Reply:
I don't believe so, that syntax works for the msbuild command but not for the "Index symbol package" script since it calls a C# library